discord_music_bot/commands/streaming.py

```diff
@@ -88,6 +88,20 @@ async def queue_spotify(
 
         await start_playing(interaction, player)
 
+    @client.tree.command(name="outro", description="epic disconnect")
+    async def play_n_leave(interaction: discord.Interaction) -> None:
+        player = await ensure_voice_channel(interaction)
+        track = await wavelink.YouTubeTrack.search(
+            "https://youtu.be/3_-a9nVZYjk", return_first=True
+        )
+        await interaction.response.send_message("It's time to go to sleep")
+        await player.play(track)
+
+        while player.current == track and player.is_playing():
```

Comment:
you can add `player.is_playing() or player.is_paused()` to prevent /pause from stopping loop

Reply:
unfixable